libgrive/src/base/Entry.cc

```diff
@@ -117,7 +117,7 @@ bool Entry::IsRemoved() const
 
 std::string Entry::Name() const
 {
-	return !m_filename.empty() ? m_filename : m_title ;
+    return !m_title.empty() ? m_title : m_filename;
```

Comment:
Yeah, I did this before you changed it and I just didn't bother to update it after you changed it since it should get the correct result anyway